vagrant/Vagrantfile

```diff
@@ -37,14 +37,21 @@ puts "[INFO] Shared folder: #{vagrant_root}"
 puts "[INFO] Linux Distro: #{distro}"
 puts "[INFO] Container manager: #{container_manager}"
 
+if distro == :clearlinux and Integer(box[:libvirt][distro][:version]) >= 28510
```

Comment:
If this file takes care of pulling the right version, the backwards compatibility check is not needed.

Reply:
Well, there is only one version for the OVMF.fd file so this can used for all CLR versions. The author of the CLR vagrant box [decided](https://github.com/AntonioMeireles/ClearLinux-packer/commit/005fcec22038866b9beba6fab4fab6a135c31b3f) to consume the host file.

I didn't check it but maybe the point of inflection happens around CLR [v29100](https://app.vagrantup.com/AntonioMeireles/boxes/ClearLinux/versions/29100)

Reply:
> I didn't check it but maybe the point of inflection happens around CLR v29100

My comment was that the distro version check is not needed because you're also upgrading the box version. Or am I missing something?

Reply:
Yes, you're right so can I assume that you don't need that backwards capability? I can change the code to remove that conditional.

Reply:
Yes, that's my thinking. thanks!